bindata/network/ovn-kubernetes/common/004-rbac-control-plane.yaml

```diff
@@ -173,6 +173,24 @@ rules:
     - services/finalizers
   verbs:
     - update
+- apiGroups: ["k8s.ovn.org"]
+  resources:
+    - userdefinednetworks
+    - userdefinednetworks/status
+  verbs:
+    - get
+    - list
+    - patch
+    - update
```

Comment:
I think thins should be a separate group, that is applied to both `userdefinednetworks` and `userdefinednetworks/status` as get/list/watch doesn't make sense on /status subresource. Check `ipamclaims` for an example

Reply:
Done